libs/platform/src/lib/components/link/link.component.html

```diff
@@ -0,0 +1,16 @@
+<a
+    #link
+    class="fd-link"
+    [attr.id]="id"
+    [attr.href]="href"
+    [target]="target"
+    [attr.aria-disabled]="disabled"
+    [ngClass]="{ 'fd-link--emphasized': isEmphasized, 'fd-link--inverted': inverted, 'fdp-link-truncate__txt': !wrap }"
+    (click)="clicked($event)"
+    [ngStyle]="{ width: width }"
+    (focus)="onFocusEvent()"
+    (focusout)="onFocusOutEvent()"
+>
+    <span class="tooltiptext" *ngIf="toolTipText && tooltipVisibility">{{ toolTipText }}</span>
```

Comment:
Can you help out with this ?  the toolTipText? What is it good for? Why we can no use native functionality of the achor that will also works with ARIA . 

Please check out ariba-label, label. ariba-title, title. I dont thing we need this. 

Also I dont see icon support here. When we make the component to beinstantiated programatically and placed to the view I woudl expect that I am able to bind all these things. 

the only thing that will stay in the ng-content is actual text, icons positioning it should be something to be handled by component

Reply:
now implemented toolTip like <a [title]="title text">.

Q. for icon support do i need to put <fd-icon> like below.
<a [title]=".."> 
  <fd-icon [glyph]="icon" [size]="iconSize" *ngIf="icon && iconBefore"></fd-icon>
  <ng-content></ng-content>
  <fd-icon [glyph]="icon" [size]="iconSize" *ngIf="icon && iconAfter"></fd-icon>
</a>

or it will be passed as a content.
<fdp-link [href]=""  [title]="'tooltip text showing on focus'" [disabled]="true">
        <fd-icon [glyph]="'chain-link'" [size]="'l'"></fd-icon>
         Link Text
</fdp-link>.
If it is later case. then how position will get handled by component?

Reply:
implemented later example show above.